hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/utils/LegacyHadoopConfigurationSource.java

```diff
@@ -76,6 +164,12 @@ public static Configuration asHadoopConfiguration(
     }
   }
 
+  /**
+   * Use this with beware, as we are returning the Configuration object, without compliance checks.
+   * Currently it's only used in the OzoneConfiguration class, in an OzoneConfiguration constructor,
+   * so we'll handle the compliance checks in the OzoneConfiguration object.
+   * @return
+   */
   public Configuration getOriginalHadoopConfiguration() {
```

Comment:
Because of https://github.com/apache/ozone/pull/6860#discussion_r1672385301 I removed this javadoc, as we will return a Configuration that also has the compliance check (even though it's still a Configuration object, not OzoneConfiguration)